main.css

```diff
@@ -11,9 +11,15 @@
     font-weight: 600;
 }
 
+.module.MMM-MonthCalendar .module-header {
+    text-align: center;
+}
+
```

Comment:
Not center-aligning the header was actually intentional, to keep it inline with the other modules. Depending on the region where the module is displayed, the header will be aligned accordingly by the global css. IMO users should override this behavior in their custom.css if they want to do so. WDYT?

Reply:
I removed the centering to adhere to the global style